lib/msf/base/serializer/opt_conditions.rb

```diff
@@ -0,0 +1,52 @@
+# -*- coding: binary -*-
+module Msf
+  module Serializer
```

Comment:
Unless the plan is to only ever use this for showing options and not determining whether we should validate options or show them during tab completion, for example, I'd add this as to `lib/msf/core/opt_base.rb` or as `lib/msf/core/opt_condition.rb`.

Reply:
Additionally to showing option we may add the validation of the options and show them during tab completion, so I sent the file to lib/msf/core/opt_condition.rb. 
For the file header I used:
```
module Msf
  module OptConditions
```
is that correct?

Reply:
For things to work smoothly, the filename should match the class or module inside, so it should be `module OptCondition`.